src/openforms/forms/tests/test_data_migrations.py

```diff
@@ -521,3 +527,62 @@ def test_open_when_empty_is_false(self):
             component = components["editgrid4"]
 
             self.assertFalse(component["openWhenEmpty"])
+
+
+class FormVariableMigrationTests(TestCase):
+    app = "forms"
+
+    def setUp(self):
+        self.form_definition = FormDefinition.objects.create(
+            name="FormDefinition1",
+            configuration={
+                "components": [
+                    {
+                        "key": "theFirstName",
+                        "type": "textfield",
+                        "label": "First Name",
+                        "prefill": {
+                            "plugin": "demo",
+                            "attribute": "",
+                        },
+                    },
+                ]
+            }
+        )
+        self.form = Form.objects.create(name="Form1")
+        self.formstep = FormStep.objects.create(
+            form=self.form, form_definition=self.form_definition
+        )
+        self.form_definition2 = FormDefinition.objects.create(
+            name="FormDefinition2",
+            configuration={
+                "components": [
+                    {
+                        "key": "theFirstName",
+                        "type": "textfield",
+                        "label": "First Name",
+                        "prefill": {
+                            "plugin": "",
+                            "attribute": "voornamen",
+                        },
+                    },
+                ]
+            }
+        )
+        self.form2 = Form.objects.create(name="Form2")
+        self.formstep2 = FormStep.objects.create(
+            form=self.form2, form_definition=self.form_definition2
+        )
+
+    def test_migrate(self):
+        """import and test function from squashed migration 003..."""
+        test_func = import_string(
+            "openforms.forms.migrations."
+            "0033_formvariable_datamigration.create_form_variables_for_form"
+        )
+
+        test_func(apps, schema_editor=None)
+        form_variables = FormVariable.objects.all()
```

Comment:
```suggestion
        test_func(apps, schema_editor=None)
        
        form_variables = FormVariable.objects.all()
```
we prefer to visually separate the set up / call-test-subject / assertions blocks with a blank line